### ISSUE TITLE
about create playlist automatic

### ISSUE THREAD

**Issue by pc10201** [porjo/youtubeuploader#45]
If I upload a video with an playlist which doesn't exist
It tell me error
```
2018/05/19 20:17:46 Error adding video to playlist: playlist ID xxxx doesn't exist
```

Can you create playlist automatic when it doesn't exist?

**Comment by porjo** [comment]
That is not currently supported, however could be added. Unless youtubeuploader is also modified to handle multiple uploads in a single call (putting them all into the new playlist) I'm not sure how useful it would be... What use case did you have in mind?

**Comment by pc10201** [comment]
Like this repo

https://github.com/tokland/youtube-upload

It can upload a video to a playlist (if not found, it will be created)

**Comment by porjo** [comment]
I've published [version 18.11](https://github.com/porjo/youtubeuploader/releases/tag/18.11) which handles creation of playlists. Include `playlistTitles` entry in the meta JSON file (see [README](https://github.com/porjo/youtubeuploader#metadata))

**Comment by pc10201** [comment]
It can create playlist now.

But It have another bug.

I set my video privacyStatus to public,but the program will create a private playlist,not public

**Comment by porjo** [comment]
Fixed in [18.12](https://github.com/porjo/youtubeuploader/releases/tag/18.12) - please try again

**Comment by pc10201** [comment]
It works.Thanks a lot.
